README.md

```diff
@@ -2,7 +2,7 @@
 
```

Comment:
- [ ] **Optional:** It is a conventional practice to add a license or reference that is why[ Microverse has provided you with one in their template](https://github.com/microverseinc/readme-template). Consider including it in future projects.